### PR TITLE
manually set document updatedAt when child fields created/updated/deleted/reordered

### PR DESCRIPTION
**Pull request by jjidt**

### CODE REVIEW

**Comment by jcontonio** [comment]
Works nicely for me 👍 
@dheniges can you take a look since we both have code here?

**Comment by jjidt** [comment]
fixed the updatedAt toolbar issue, the fields not getting updatedAt changes on value update seems like a potential sequelize bug, so gonna leave that alone for now and make a ticket

File: web/src/client/js/reducers/documents.js [hunk 1]
```diff
@@ -152,7 +152,7 @@ export const documents = (state = initialState, action) => {
     case ActionTypes.RECEIVE_CREATED_FIELD:
     case ActionTypes.RECEIVE_UPDATED_FIELD:
     case ActionTypes.REMOVE_FIELD: {
-      const documentToUpdate = state.projectDocumentsById[action.projectId][action.documentId]
+      const documentToUpdate = { ...state.projectDocumentsById[action.projectId][action.documentId] }
```
Comment (jcontonio) [inline web/src/client/js/reducers/documents.js:155]: oh doh!

**Review by jcontonio — APPROVED** [review]
👍

**Pull request merged**